uber/models.py

```diff
@@ -565,52 +565,46 @@ def shift_badges(self, badge_type, badge_num, *, until=None, **direction):
                 a.badge_num += shift
 
         def change_badge(self, attendee, badge_type, badge_num=None):
```

Comment:
This function was heavily refactored as part of the Anthrocon branch.  On the plus side, that branch improved this by eliminating ``CUSTOM_BADGES_REALLY_ORDERED`` and replacing it with ``SHIFT_CUSTOM_BADGES``.  The downside is that several edge cases were broken by the refactor, which the unit tests revealed.

At some point we should look through this function and make sure it has literally 100% branch coverage in unit tests, since it's historically been the most brittle function in our codebase, due mainly to having so many edge cases.